source/simulator/stokes_matrix_free.cc

```diff
@@ -2342,6 +2357,22 @@ namespace aspect
       DoFTools::extract_locally_relevant_dofs (dof_handler_v,
                                                locally_relevant_dofs);
       constraints_v.reinit(locally_relevant_dofs);
+
+
+      {
+        typedef std::set< std::pair< std::pair< types::boundary_id, types::boundary_id>, unsigned int>>
```

Comment:
We dont use space at the closing angle brackets, so why have them when opening? Probably a subjective question:
```suggestion
        typedef std::set<std::pair<std::pair<types::boundary_id, types::boundary_id>, unsigned int>>
```